__testdata__/testsql.sql

```diff
@@ -0,0 +1,7 @@
+-- This script is used by pr-check.yml to test the SQLCMD action
+
+-- This should successfully insert data into the table created in the DACPAC step
+INSERT INTO [Table1] VALUES(1, 'test');
+
+-- Cleanup afterwards
+TRUNCATE TABLE [Table1];
```

Comment:
drop instead of truncate so that deploying the dacpac will recreate the table the next time the test is run if the db wasn't deleted yet?